listenbrainz/webserver/static/js/src/year-in-music/YearInMusic.tsx

```diff
@@ -356,7 +357,11 @@ export default class YearInMusic extends React.Component<
               >
                 {yearInMusicData.top_releases.slice(0, 10).map((release) => (
                   <img
-                    src="/static/img/cover-art-placeholder.jpg"
+                    src={
+                      yearInMusicData.top_releases_coverart[
+                        release.release_mbid
```

Comment:
no checking here to see if this mbid is set, or if it exists in the coverart array

Reply:
@MonkeyDo

Reply:
You can add ` ?? "/static/img/cover-art-placeholder.jpg"` to default to the placeholder image when no item is found in the coverart dict